api/resource.rb

```diff
@@ -357,6 +357,15 @@ def regex_url
                    .gsub('{{zone}}', '[a-z1-9\-]*')
     end
 
+    # All setable properties in the resource.
+    # Fingerprints aren't *really" settable properties, but they behave like one.
+    # At Create, they have no value but they can just be read in anyways, and after a Read
+    # they will need ot be set in every Update.
+    def settable_properties
+      all_user_properties.reject { |v| v.output && !v.is_a?(Api::Type::Fingerprint) }
```

Comment:
So to be clear we *do* want to include fingerprints?

Reply:
Yes! That change was me, I forget the _exact_ reasoning but fingerprint acted like a settable/updatable field at every time but `Create`.